node/cmd/guardiand/node.go

```diff
@@ -500,6 +510,8 @@ func init() {
 	gatewayRelayerKeyPassPhrase = NodeCmd.Flags().String("gatewayRelayerKeyPassPhrase", "", "Pass phrase used to unarmor the gateway relayer key file")
 
 	subscribeToVAAs = NodeCmd.Flags().Bool("subscribeToVAAs", false, "Guardiand should subscribe to incoming signed VAAs, set to true if running a public RPC node")
+
+	transferVerifierEnabledChains = NodeCmd.Flags().UintSlice("transferVerifierEnabledChains", make([]uint, 2), "Transfer Verifier will be enabled for these chain IDs (comma-separated)")
```

Comment:
What's the idea behind the default case?

Reply:
Hm my intention is that you can invoke the guardian with e.g.:

`txVerifierChains=1,2`

but not `txVerifierChains=` (blank)

If you mean the `make` here, I gave it a length and capacity of `2` more or less arbitrarily. I guess I was thinking that if we enable Ethereum and Sui, that's two chains. The value could easily be `0` or `10` or anything else.

Reply:
Couldn't you just make the default nil here and then at line 938 do `if len(*transferVerifierEnabledChains) != 0 {` instead of using Changed? Seems more clear to me.

Reply:
I think Jason implemented this in a recent commit